packages/editor/src/components/widgets/headlineControls/HeadlineControls.tsx

```diff
@@ -0,0 +1,30 @@
+import './HeadlineControls.css';
+import Button from '../button/Button';
+import type { FieldsetControl } from '../fieldset';
+
+type HeadlineControlsProps = {
+  controls: FieldsetControl[];
+  open?: boolean;
```

Comment:
the open has nothing to do with this component IMO. This is only a special case for the collapsible, that if it is open the controls are shown. So just handle it there.